ui/src/components/Header/Controller.tsx

```diff
@@ -38,22 +40,27 @@ export const Controller = (): ReactElement => {
   }, [isLoading]);
 
   const buildMountArg = () => {
-    const OSPath = ddClient.host.platform === 'darwin'
-      ? 'Users'
-      : 'home';
+    if(user === '../temp'){
```

Comment:
Can we add some comments in here to explain why/how this case can happen? Seems a bit counter-intuitive that the `user` variable can contain a relative path name..